api/app/signals/apps/reporting/tasks.py

```diff
@@ -18,3 +21,13 @@ def task_save_csv_files_datawarehouse():
     :returns:
     """
     save_csv_files_datawarehouse()
+
+
+@app.task
+def task_save_csv_files_endpoint():
+    """Celery task to save CSV files for csv endpoint
+    This task is scheduled in Celery beat to run periodically.
+
+    :returns:
+    """
+    save_csv_files_endpoint()
```

Comment:
Do we have to think about where the process boundary is, this is a task calling a task (in process in this case). This looks "drafty".

Reply:
followed the 'standard' how it's being done for other stuff, but agree that we can put it in 1